scene/resources/sky.cpp

```diff
@@ -84,7 +84,7 @@ void Sky::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "sky_material", PROPERTY_HINT_RESOURCE_TYPE, "ShaderMaterial,PanoramaSkyMaterial,ProceduralSkyMaterial,PhysicalSkyMaterial"), "set_material", "get_material");
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "process_mode", PROPERTY_HINT_ENUM, "Automatic,HighQuality,HighQualityIncremental,RealTime"), "set_process_mode", "get_process_mode");
-	ADD_PROPERTY(PropertyInfo(Variant::INT, "radiance_size", PROPERTY_HINT_ENUM, "32,64,128,256,512,1024,2048"), "set_radiance_size", "get_radiance_size");
+	ADD_PROPERTY(PropertyInfo(Variant::INT, "radiance_size", PROPERTY_HINT_ENUM, "32 (Fastest),64 (Faster),128 (Fast),256 (Average),512 (Slow),1024 (Slower),2048 (Slowest)"), "set_radiance_size", "get_radiance_size");
```

Comment:
Radiance size isn't so much a speed optimization as a memory optimization (which can impact speed in certain scenarios). 

In some scenarios 2048 may be slower than 32, but 64 would almost never be slower than 32. 

I don't think we should try to classify these by speed.

Reply:
Doesn't changing the radiance size affect the sky update speed?

Reply:
@Calinou: Yes it does affect sky update speed

Reply:
Ah, yes it does if you are updating your sky shader every frame. So it is definitely more nuanced than my message says.